schema/object.go

```diff
@@ -19,11 +23,22 @@ type Object interface {
 // NewObjectSchema creates a new object definition.
 // If you need it tied to a struct, use NewStructMappedObjectSchema instead.
 func NewObjectSchema(id string, properties map[string]*PropertySchema) *ObjectSchema {
+	return newObjectSchema(id, properties, false)
+}
+
+// NewLooseObjectSchema creates a new object definition with loose ID checking.
```

Comment:
"with unenforced ID checking"?? Same in the next sentence ...

Reply:
Fixed.